relauncher.py

```diff
@@ -1,11 +1,61 @@
 import os, time
 
+# USER CHANGABLE ARGUMENTS
+
+# Change to `True` if you wish to enable these common arguments
+
+# Run upscaling models on the CPU
+extra_models_cpu = False
+
+# Automatically open a new browser window or tab on first launch
+inbrowser = False
+
+# Run Stable Diffusion in Optimized Mode - Only requires 4Gb of VRAM, but is significantly slower
+optimized = False
+
+# Run in Optimized Turbo Mode - Needs more VRAM than regular optimized mode, but is faster
+optimized_turbo = False
+
+# Creates a public xxxxx.gradio.app share link to allow others to use your interface (requires properly forwarded ports to work correctly)
+share = False
+
+
+# Enter other `--arguments` you wish to use - Must be entered as a `--argument ` syntax
+other_arguments = ""
+
+
+
+
+
+# BEGIN RELAUNCHER PYTHON CODE
+
+common_arguments = ""
+
+if extra_models_cpu == True:
+    common_arguments += "--extra-models-cpu "
+if optimized_turbo == True:
+    common_arguments += "--optimized-turbo "
+if optimized == True:
+    common_arguments += "--optimized "
+if share == True:
+    common_arguments += "--share "
+
+if inbrowser == True:
+    inbrowser_argument = "--inbrowser "
+else:
+    inbrowser_argument = ""
+
 n = 0
 while True:
-    print('Relauncher: Launching...')
-    if n > 0:
+    if n == 0:
+        print('Relauncher: Launching...')
+        os.system(f"python scripts/webui.py {common_arguments} {inbrowser_argument} {other_arguments}")
+        
+    else:
         print(f'\tRelaunch count: {n}')

```

Comment:
While you're here, maybe let's kill this after a certain amount of retries?

Reply:
Yeah, absolutely. What do you think is a reasonable number? 15? 20 maybe? Don't want to exit out unless there's truly an unresolvable starting loop.

Reply:
100?

Reply:
LOL! Ok, I can set it to that!